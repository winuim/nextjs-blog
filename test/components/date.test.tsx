import React from "react";
import { render } from "../testUtils";
import Date from "../../components/date";

describe("Home page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Date dateString={"2021-01-01 "} />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
