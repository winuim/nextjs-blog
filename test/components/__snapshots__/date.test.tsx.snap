// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <time
    datetime="2021-01-01 "
  >
    January 1, 2021
  </time>
</DocumentFragment>
`;
